src/World.cpp

```diff
@@ -624,6 +624,12 @@ void cWorld::GenerateRandomSpawn(void)
 
 bool cWorld::CheckPlayerSpawnPoint(int a_PosX, int a_PosY, int a_PosZ)
 {
+	// The bottom layer cannot hold a valid spawn point
+	if (a_PosY <= 0)
+	{
+		return false;
+	}
+	
```

Comment:
I believe we keep the tabs on those lines. At least it's still [here](https://github.com/cuberite/cuberite/blob/master/src/World.cpp#L2556), [here](https://github.com/cuberite/cuberite/blob/master/src/World.cpp#L2733), and [here](https://github.com/cuberite/cuberite/blob/master/src/World.cpp#L3089)